config/routes.rb

```diff
@@ -10,4 +10,8 @@
     )
   end
 
+  resources :errors, only: [:show]
+
+  # See http://patrickperey.com/railscast-053-handling-exceptions/
+  get "(errors)/:id", to: "errors#show"
```

Comment:
This is similar to WEX but I've used `:id` rather than `:status` so that the show action behaves as normal. When rack generates an error it sends it to `errors/<http_status>`, so the status is passed through to the action as the `:id`. That means you don't have to dig around within the exception to get the status as WEX does.